docs/faq/code-analysis/which-metrics-does-codacy-calculate.md

```diff
@@ -0,0 +1,124 @@
+---
+description: Codacy scans your code for issues and calculates the metrics code complexity, duplication, and coverage. Besides this, Codacy also calculates a grade for your repository and files based on all calculated code quality metrics.
```

Comment:
```suggestion
description: Codacy scans your code for issues and calculates code complexity, duplication, and coverage metrics. Besides this, Codacy also calculates a grade for your repository and files based on all calculated code quality metrics.
```

Reply:
Nice suggestion, it improves the flow of the text!